pkg/monitoring/clusterroles_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package monitoring
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+)
+
+func TestGetSystemClusterRoles(t *testing.T) {
+	clusterName := "test-cluster"
```

Comment:
If you do the following then you don't need to pass t to assert
	assert := assert.New(t)

Reply:
nice, I will make that adjustment to this text in the next unit test I am adding